codegen/src/main/java/com/paypal/selion/elements/HtmlSeLionElement.java

```diff
@@ -25,20 +26,20 @@
  *
  */
 public class HtmlSeLionElement extends AbstractSeLionElement {
-    public static HtmlSeLionElement TEXT_FIELD = new HtmlSeLionElement("TextField", true, true);
-    public static HtmlSeLionElement TABLE = new HtmlSeLionElement("Table", true, true);
-    public static HtmlSeLionElement SELECT_LIST = new HtmlSeLionElement("SelectList", true, true);
-    public static HtmlSeLionElement RADIO_BUTTON = new HtmlSeLionElement("RadioButton", true, true);
-    public static HtmlSeLionElement BUTTON = new HtmlSeLionElement("Button", true, true);
-    public static HtmlSeLionElement LINK = new HtmlSeLionElement("Link", true, true);
-    public static HtmlSeLionElement LABEL = new HtmlSeLionElement("Label", true, true);
-    public static HtmlSeLionElement IMAGE = new HtmlSeLionElement("Image", true, true);
-    public static HtmlSeLionElement FORM = new HtmlSeLionElement("Form", true, true);
-    public static HtmlSeLionElement DATE_PICKER = new HtmlSeLionElement("DatePicker", true, true);
-    public static HtmlSeLionElement CHECK_BOX = new HtmlSeLionElement("CheckBox", true, true);
-    public static HtmlSeLionElement CONTAINER = new HtmlSeLionElement("Container", true, false);
-    public static HtmlSeLionElement BASE_CLASS = new HtmlSeLionElement("baseClass", false, false);
-    public static HtmlSeLionElement PAGE_TITLE = new HtmlSeLionElement("pageTitle", false, false);
+    public static HtmlSeLionElement TEXT_FIELD = new HtmlSeLionElement("com.paypal.selion.platform.html", "TextField", true, true);
```

Comment:
Do you mean in the class AbstractSeLionElement?

```
private String element;
private boolean uiElement;
private String elementPackage;
```

This is how the class looks now so I kept it the same, you want me to update all to public static final?